contracts/LpSugar.vy

```diff
@@ -1106,8 +1106,9 @@ def _safe_symbol(_token: address) -> String[100]:
   )[1]
 
   # Check response as revert_on_failure is set to False
-  if len(response) > 0:
-    return abi_decode(response, String[100])
+  # String size cannot be larger than max_outsize - 64 bytes
+  if len(response) > 0 and len(response) <= 128:
+    return abi_decode(response, String[128])
```

Comment:
My understanding is that this should also be a `String[192]` if we're doing the `max_outsize=192`, no?